src/test/java/AcmeShop.java

```diff
@@ -114,10 +107,105 @@ public void addingToCart() {
+        allNames.add(pinkDuck);
+
+        ArrayList<String> forNamesComparison = new ArrayList<>();
+        forNamesComparison.add("Green DucK");
+        forNamesComparison.add("Yellow Duck");
+        forNamesComparison.add("Розовая уточка");
+
+        Assert.assertEquals(allNames, forNamesComparison);
+
+        //Сортировка по цене
+        WebElement priceButton = webdriver.findElement(By.linkText("Price"));
+        priceButton.click();
+
+        WebElement yellowPrice = webdriver.findElement(By.cssSelector(".link[href=\"https://litecart.stqa.ru/en/rubber-ducks-c-1/subcategory-c-2/yellow-duck-p-1\"] strong[class*=price]"));
+        String yellowPriceString = yellowPrice.getText();
+        String yellowPriceTruncated = yellowPriceString.substring(0, yellowPriceString.length() - 2); // обрезаем пробел и значок валюты
+        double yellowPriceDouble = Double.parseDouble(yellowPriceTruncated);
+
+        WebElement greenPrice = webdriver.findElement(By.cssSelector(".link[href=\"https://litecart.stqa.ru/en/rubber-ducks-c-1/subcategory-c-2/green-duck-p-2\"] span[class*=price]"));
+        String greenPriceString = greenPrice.getText();
+        String greenPriceTruncated = greenPriceString.substring(0, greenPriceString.length() - 2); // обрезаем пробел и значок валюты
+        double greenPriceDouble = Double.parseDouble(greenPriceTruncated);
+
+        WebElement pinkPrice = webdriver.findElement(By.cssSelector(".link[href=\"https://litecart.stqa.ru/en/rubber-ducks-c-1/subcategory-c-2/rozovaya-utochka-p-6\"] span[class*=price]"));
+        String pinkPriceString = pinkPrice.getText();
+        String pinkPriceTruncated = pinkPriceString.substring(0, pinkPriceString.length() - 2); // обрезаем пробел и значок валюты
+        double pinkPriceDouble = Double.parseDouble(pinkPriceTruncated);
+
+        ArrayList<Double> allPrices = new ArrayList<>();
+        allPrices.add(yellowPriceDouble);
+        allPrices.add(greenPriceDouble);
+        allPrices.add(pinkPriceDouble);
+
+        String result;
+        if (allPrices.get(0) < allPrices.get(1) && allPrices.get(1) < allPrices.get(2)) {
+            result = "Sorted";
+        } else {
+            result = "Not Sorted";
+        }
+
+        Assert.assertEquals(result, "Sorted");
+        }
+
+        @Test //Проверка соответствия лэйблов
+        public void labelsChecking() {
+        WebElement rubberDucksHeader = webdriver.findElement(By.cssSelector("#site-menu-wrapper .category-1"));
+        rubberDucksHeader.click();
+
+        WebElement subcategoryLink = webdriver.findElement(By.linkText("Subcategory"));
+        subcategoryLink.click();
 
+        WebElement greenDuckLabel = webdriver.findElement(By.
+                cssSelector(".link[href=\"https://litecart.stqa.ru/en/rubber-ducks-c-1/subcategory-c-2/green-duck-p-2\"] div[class*=\"sticker\"]"));
+        String greenDuckLabelString = greenDuckLabel.getText();
+        Assert.assertEquals(greenDuckLabelString, "NEW");
```

Comment:
Please use SoftAssert here and in other validations of this method or even better - split into separate tests